lib/matplotlib/cbook/__init__.py

```diff
@@ -1579,6 +1579,48 @@ def simple_linear_interpolation(a, steps):
             .reshape((len(x),) + a.shape[1:]))
 
 
+def less_simple_linear_interpolation(x, y, xi, extrap=False):
```

Comment:
I deprecated this in mlab, so I guess it can stay there...

Reply:
As in I should move it back to mlab in this PR?

Reply:
Yes